src/jquery.popup-interactive-world-map.js

```diff
@@ -29,7 +29,10 @@
 
             $element     = $('g', settings.element).find("[data-code='" + settings.code  + "']");
             $offset      = $element.offset();
-            $modal = $("." + settings.classes.modal );
+
+            $context = $element.parent().parent().parent().parent();
```

Comment:
Bascially yes, but take a look at https://api.jquery.com/parentsUntil or https://api.jquery.com/closest to make it a bit prettier :wink: